app/assets/javascripts/polaris_view_components/popover_controller.js

```diff
@@ -1,9 +1,9 @@
-import { Controller } from "@hotwired/stimulus"
-import { createPopper } from "@popperjs/core/dist/esm"
+import { Controller } from '@hotwired/stimulus'
+import { createPopper } from '@popperjs/core/dist/esm'
```

Comment:
I used Prettier to automatically format this. I can share `.prettierrc` that I used if you'd like?

Reply:
Here it is actually:

```json
{
  "printWidth": 120,
  "singleQuote": true,
  "semi": false,
  "arrowParens": "avoid",
  "trailingComma": "none"
}
```

Reply:
Let's exclude these changes from the PR. They will just add inconsistency in JS code style at this point. We use [Standard RB](https://github.com/testdouble/standard) for Ruby code style. It would be great to have [Standard JS](https://standardjs.com/) too. But it's for another PR 🙂